src/components/directory-content.js

```diff
@@ -599,19 +611,18 @@ const DirectoryContent = () => {
                         childrenMetadata[elementUuid].subtype
                     )}
                 {/* Name */}
-                {isMetadataLoading ? null : childrenMetadata[elementUuid] ? (
-                    <div>{elementName}</div>
-                ) : (
-                    <>
-                        {elementName + ' '}
-                        <FormattedMessage id="creationInProgress" />
-                    </>
-                )}
+                {<div>{getDisplayedElementName(cellData)}</div>}
             </div>
         );
     };
 
     function toggleSelection(elementUuid) {
+        let element = currentChildren?.find(
+            (e) => e.elementUuid === elementUuid
+        );
+        if (element.uploading) {
```

Comment:
No protection against absence from currentChildren ? (Does it make sense, either ? I have not checked)